Controller/DashboardController.php

```diff
@@ -30,6 +30,10 @@ class DashboardController extends BaseController
      */
     public function index(Request $request, ReminderRepository $reminderRepository, ProcessRepository $processRepository, PaginatorInterface $paginator, SessionInterface $session, ProcessManager $processManager)
     {
+        if ($this->isGranted('ROLE_EXTERNAL')) {
```

Comment:
Naming the role `ROLE_VIEW` (or `ROLE_PROCESS_VIEW`), say, may make it more clear what the role actually allows the user to do.

Reply:
Role is renamed.